packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -1087,6 +1122,28 @@ export class Viewer implements IDisposable {
         this._details.model?.animationGroups.forEach((group) => (group.speedRatio = this._animationSpeed));
     }
 
+    private _pick(screenX: number, screenY: number): Nullable<PickingInfo> {
+        if (this._details.model) {
+            const model = this._details.model;
+            // Refresh bounding info to ensure morph target and skeletal animations are taken into account.
+            model.meshes.forEach((mesh) => {

```

Comment:
Should this be only updated if it has rendered ? or should the doc mention to be careful with high frequency calls (like pointer move) ?

Reply:
Hmm good question, I didn't think about the possibility of someone calling this pick function in a pointer move (or some other high frequency code path). I would say for now let's just document it (I will update the doc comments) and see if this is a real scenario that comes up with any viewer usage.